client/multi-currency/enabled-currencies-list/modal-checkbox.js

```diff
@@ -5,13 +5,13 @@
 import React from 'react';
 import { CheckboxControl } from '@wordpress/components';
 import { useCallback } from '@wordpress/element';
+import interpolateComponents from 'interpolate-components';
 
+// TODO: The currency.symbol may be a HTML element, so that needs to be fixed.
```

Comment:
Just to confirm, am I right in thinking this is being done as part of #1922?

Reply:
Yup, and the PR that addresses the issue removes that TODO: https://github.com/Automattic/woocommerce-payments/pull/1930/files#diff-993f704effe6480bba28b91407b5e87fbffd86a894528d7c108d142063d96255L10